modules/influxdb/src/test/java/org/testcontainers/containers/InfluxDBTestUtils.java

```diff
@@ -7,4 +7,6 @@ public final class InfluxDBTestUtils {
     static final DockerImageName INFLUXDB_V1_TEST_IMAGE = DockerImageName.parse("influxdb:1.4.3");
 
     static final DockerImageName INFLUXDB_V2_TEST_IMAGE = DockerImageName.parse("influxdb:2.0.7");
+
+    public static final DockerImageName INFLUXDB_V3_TEST_IMAGE = DockerImageName.parse("influxdb:3-core");
```

Comment:
use a fixed version. `influxdb:3-core` is a mutable tag, it will always point to latest 3.x version, let's avoid that.